### PR TITLE
Merchant button fixes #5446

### LINKED ISSUE #5446: Button in Alexander the Merchant popover is too long (with certain strings)
Originally reported in [#5209](https://github.com/HabitRPG/habitrpg/issues/5209#issuecomment-112869188).

a

selection_034
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/22653688-button-in-alexander-the-merchant-popover-is-too-long-with-certain-strings?utm_campaign=plugin&utm_content=tracker%2F68393&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F68393&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Thrashmandicoot**
https://github.com/HabitRPG/habitrpg/issues/5446

Refactors merchant sales
-adds item to display in merchant box for easy identification
-removes item name from sale button, so sale button is consistent

### CODE REVIEW

**Comment by Alys** [comment]
Could you please show us screenshot(s)? Or link to the shot in the issue if it's still the same.

**Comment by Thrashmandicoot** [comment]
Here you go!

screenshot from 2015-10-30 03 59 43
screenshot from 2015-10-30 03 59 47
screenshot from 2015-10-30 03 59 53
screenshot from 2015-10-30 04 00 00
screenshot from 2015-10-30 04 00 06

**Comment by Alys** [comment]
I like it! 

@lemoness Is this okay with you?

File: website/views/options/inventory/drops.jade [hunk 1]
```diff
@@ -93,10 +93,22 @@
             a(target='_blank', href='http://www.kickstarter.com/profile/523661924')=env.t('alexander')
           .popover-content
             p=env.t('welcomeMarket')
-            p
-              button.btn.btn-primary(ng-show='selectedEgg', ng-click='sellInventory()')=env.t('sellEggForGold', {itemType: "{{selectedEgg.text()}}", gold: "{{selectedEgg.value}}"})
-              button.btn.btn-primary(ng-show='selectedPotion', ng-click='sellInventory()')=env.t('sellPotionForGold', {itemType: "{{selectedPotion.text()}}", gold: "{{selectedPotion.value}}"})
-              button.btn.btn-primary(ng-show='selectedFood', ng-click='sellInventory()')=env.t('sellForGold', {item: "{{selectedFood.text()}}", gold: "{{selectedFood.value}}"})
+            hr(ng-show='selectedEgg || selectedPotion || selectedFood')
+            div(ng-show='selectedEgg || selectedPotion || selectedFood')
+              div.customize-option(class='Pet_Egg_{{selectedEgg.key}}' style='float:left;' ng-show='selectedEgg')
```
Comment (crookedneighbor) [inline website/views/options/inventory/drops.jade:98]: You should use the `.pull-left` class instead of inline styles.

Comment (Thrashmandicoot) [inline website/views/options/inventory/drops.jade:98]: @crookedneighbor agreed, I'll fix that.

Comment (Thrashmandicoot) [inline website/views/options/inventory/drops.jade:98]: @crookedneighbor fixed!

**Comment by Thrashmandicoot** [comment]
Added class styling, removed inline styling.

**Comment by crookedneighbor** [comment]
Made some final adjustments: https://github.com/HabitRPG/habitrpg/commit/92271c20491b6295400070a0d526f39b691361ea

This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by Thrashmandicoot** [comment]
@crookedneighbor User ID: fb3819d0-344f-46f0-a10c-cb8cbbeafa4c

**Comment by crookedneighbor** [comment]
Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Pull request merged**